src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -744,8 +753,17 @@ private void Dispose()
                 string bkptno = results.Results.FindString("bkptno");
                 ulong addr = cxt.pc ?? 0;
 
-                bool fContinue;
                 TupleValue frame = results.Results.TryFind<TupleValue>("frame");
+                if (frame != null && addr != 0)
+                {
+                    string sourceFile = frame.TryFindString("fullname");
+                    if (!String.IsNullOrEmpty(sourceFile))
+                    {
+                        await this.VerifySourceFileTimestamp(addr, sourceFile);
```

Comment:
I don't think you want to do this check unless the debugger is really going to stop. Maybe delay this check until the decision to send up a stopping event is made.